src/odemis/driver/xt_client.py

```diff
@@ -1116,18 +1122,9 @@ def __init__(self, name, role, parent, rng=None, **kwargs):
     def _updatePosition(self, raw_pos=None):
         """
         update the position VA
-        raw_pos (dict str -> float): the position in mm (as received from the SEM)
+        raw_pos (dict str -> float): the position (as received from the SEM)
```

Comment:
Could you indicate that if None is passed, it'll ask the current position to the SEM?